federation/v1.0/federation-v1.0.graphql

```diff
@@ -2,4 +2,5 @@ directive @key(fields: _FieldSet!) repeatable on OBJECT | INTERFACE
 directive @requires(fields: _FieldSet!) on FIELD_DEFINITION
 directive @provides(fields: _FieldSet!) on FIELD_DEFINITION
 directive @external on FIELD_DEFINITION
-scalar _FieldSet
\ No newline at end of file
+directive @extends on OBJECT | INTERFACE
```

Comment:
Maybe a description stating this one is optional for supporting use cases that can't use `extend`?

Reply:
we don't specify that it in v2 specs but can add it in